content/intro-to-storybook/react/fr/test.md

```diff
@@ -1,22 +1,24 @@
 ---
-title: 'Tester les composantes UI'
-tocTitle: 'Test'
+title: 'Tests visuels'
+tocTitle: 'Test visuel'
 description: 'Apprenez à tester les composants UI'
 ---
 
-Aucun tutoriel de Storybook ne serait complet sans test. Les tests sont essentiels pour créer des UI de haute qualité. Dans les systèmes modulaires, de minuscules modifications peuvent entraîner des régressions majeures. Jusqu'à présent, nous avons rencontré trois types de tests :
+Aucun tutoriel de Storybook ne serait complet sans tests. Les tests sont essentiels pour créer des UI de haute qualité. Dans les systèmes modulaires, de minuscules modifications peuvent entraîner des régressions majeures. Jusqu'à présent, nous avons rencontré trois types de tests:
 
-- **Tests manuels** qui consistent pour les développeurs à examiner manuellement un composant pour en vérifier l'exactitude. Ils nous aident à vérifier l'aspect d'un composant au fur et à mesure de sa construction.
-- **Les captures instantanées** avec Storyshots capturent le balisage rendu d'un composant. Ils nous aident à nous tenir au courant des changements de balisage qui provoquent des erreurs de rendu et des avertissements.
-- **Les tests unitaires** avec Jest vérifient que la sortie d'un composant reste la même avec une entrée fixe. Ils sont parfaits pour tester les qualités fonctionnelles d'un composant.
+- **Les tests manuels** qui consistent pour les développeurs à examiner manuellement un composant pour en vérifier l'exactitude. Ils nous aident à vérifier l'aspect d'un composant au fur et à mesure de sa construction.
 
-## "Mais est-ce que ça a l'air bien ?"
+- **Les tests d'accessibilité**, avec l'addonn a11y, vérifient que le composant est accessible à tout type de personne. Ils sont parfaits pour collecter de l'information sur comment les personnes avec des difficultés physique utilisent les composants.
```

Comment:
@xaviermarchal seems that there's an extra "n" here. Shouldn't this be "avec l'addon a11y"?

Reply:
good catch, it's removed :)